gamemode/load_assets.lua

```diff
@@ -0,0 +1,16 @@
+local function runRecurse( dir, ext, f )
+    local files, dirs = file.Find( dir .. "/*", "GAME" )
+    if not files then return end
```

Comment:
`file.Find` returns nil if the path is not valid. Does an empty table count as a falsey value?

Reply:
it does not, only nil and false are falsey in lua